supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift-container-storage.adoc

```diff
@@ -0,0 +1,145 @@
+==== Container Storage Interface (noun)
+*Description*: The Container Storage Interface (CSI) is a standard for exposing arbitrary block and file storage systems to containerized workloads on Container Orchestration Systems like Kubernetes, and in particular Red Hat OpenShift Container Platform. This allows OpenShift Container Platform to consume storage from third-party storage providers that implement the CSI interface as persistent storage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[multicloud-object-gateway]]
+==== Multicloud Object Gateway (noun)
+*Description*: Multicloud Object Gateway (MCG) is a lightweight object storage service for OpenShift Container Platform that allows users to start small and then scale as needed on-premise, in multiple clusters, and with cloud-native storage.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[namespace-bucket]]
+==== namespace bucket (noun)
+*Description*: Namespace buckets enable connecting data repositories on different providers together so that the interaction with data is possible through a single unified view. Data written to namespace buckets are kept plain and not deduped, compressed, or encrypted. The data can be consumed directly from the repository.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[namespace-store]]
+==== namespace store (noun)
+*Description*: A type of a resource for Multicloud Object Gateway that is used to store plain data and is used by namespace buckets. The supported stores are S3, Azure, and S3 compatible.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[object-bucket]]
+==== object bucket (noun)
+*Description*: A container for storing objects.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[object-bucket-claim]]
+==== object bucket claim (noun)
+*Description*: Creates a new bucket and an application account in Multicloud Object Gateway or RADOS Gateway (RGW) with permissions to the bucket, including a new access key and secret access key. An object bucket claim can be used to request an S3 compatible bucket for the workloads.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[persistent-volume]]
+==== persistent volume (noun)
+*Description*: A persistent volume (PV) is a piece of storage in the cluster that has been provisioned by an administrator or dynamically provisioned using storage classes. 
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[persistent-volume-claim]]
+==== persistent volume claim (noun)
+*Description*: A persistent volume claim (PVC) is a request for storage by a user.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[red-hat-openshift-container-storage]]
+==== Red Hat OpenShift Container Storage (noun)
+*Description*: Red Hat's software-defined storage for containers that helps to develop and deploy applications quickly and efficiently across cloud platforms.
```

Comment:
From the ISG: "Do not use 's to show the possessive form of brand names or product names."
I guess in this case, you could say: `Red Hat software-defined storage for..`

Reply:
Ack. Changed.